styles.css

```diff
@@ -1,3 +1,335 @@
-* {
```

Comment:
Set common font-size, line-height, color for text on all pages for `body` tag. 
It will default styles for text without classes.

Reply:
pls check this comment again